src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/RefreshConstants.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using System;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+    internal class RefreshConstants
+    {
+        // Key-values
+        public static readonly TimeSpan DefaultCacheExpirationInterval = TimeSpan.FromSeconds(30);
+        public static readonly TimeSpan MinimumCacheExpirationInterval = TimeSpan.FromMilliseconds(1000);
+
+        // Feature flags
+        public static readonly TimeSpan DefaultFeatureFlagsCacheExpirationInterval = TimeSpan.FromSeconds(30);
+        public static readonly TimeSpan MinimumFeatureFlagsCacheExpirationInterval = TimeSpan.FromMilliseconds(1000);
+
+        // Key Vault secrets
+        public static readonly TimeSpan MinimumSecretRefreshInterval = TimeSpan.FromMilliseconds(1000);
```

Comment:
`MinimumSecretRefreshInterval` is not being used yet. I'll create a separate PR for adding this check in KeyVault options. Since its technically a breaking change, we can release it with 5.0.0.